module/zfs/qat_compress.c

```diff
@@ -104,7 +104,7 @@ static kstat_t *qat_ksp;
 static CpaInstanceHandle dc_inst_handles[MAX_INSTANCES];
 static CpaDcSessionHandle session_handles[MAX_INSTANCES];
 static CpaBufferList **buffer_array[MAX_INSTANCES];
-static Cpa32U num_inst = 0;
+static Cpa16U num_inst = 0;
```

Comment:
The type here was previously changed from `Cpa16U` to `Cpa32U` in commit 7a25f0891eef4adbe00dd03e26b6128dc99e170d due to an overflow issue.

Reply:
the overflow is in "inst_num", the commit 7a25f08 changed "num_inst" also to keep consistent, but it causes a build warning when compile with QAT driver (head file).

Reply:
Ahh yes, OK thanks for the clarification.